web/src/features/fba/components/FBAMap.tsx

```diff
@@ -65,32 +50,34 @@ const FBAMap = (props: FBAMapProps) => {
   const [map, setMap] = useState<ol.Map | null>(null)
   const mapRef = useRef<HTMLDivElement | null>(null)
 
-  const fireCenterSource = buildFireVectorSource(fireCenterLayer)
+  const fireZoneVector = new VectorTileLayer({
+    opacity: 0.5,
+    source: new VectorTileSource({
+      attributions: 'BC Fire Zones',
+      format: new MVT(),
+      url: 'https://tileserv-dev.apps.silver.devops.gov.bc.ca/public.fire_zones/{z}/{x}/{y}.pbf'
+    }),
+    style: fireZoneStyler,
+    declutter: true
+  })
 
-  const fireCenterVector = new OLVectorLayer({
-    source: fireCenterSource,
-    style: () => {
-      return new Style({
-        stroke: new Stroke({
-          lineDash: [1, 10],
-          color: 'blue',
-          width: 4
-        })
-      })
-    }
+  const fireCenterVector = new VectorTileLayer({
+    source: new VectorTileSource({
+      attributions: 'BC Fire Centers',
+      format: new MVT(),
+      url: 'https://tileserv-dev.apps.silver.devops.gov.bc.ca/public.fire_centres/{z}/{x}/{y}.pbf'
+    }),
+    style: fireCenterStyler,
+    declutter: true
   })
 
-  const fireZoneVector = new OLVectorLayer({
-    opacity: 0.5,
-    source: buildFireVectorSource(fireZoneLayer),
-    style: () => {
-      return new Style({
-        stroke: new Stroke({
-          color: 'purple',
-          width: 4
-        })
-      })
-    }
+  const thesianVector = new VectorTileLayer({
```

Comment:
```suggestion
  const thessianVector = new VectorTileLayer({
```

Reply:
Making this spelling consistent with others, but according to Google, "thessian" should actually be spelled "thiessen". My personal preference would be to correct the spelling to thiessen, but if it's too much work, it'll probably only bother me and no one else.

Reply:
It bothers me now!